### PR TITLE
Create I2C module document.

### PR DESCRIPTION
**Pull request by nova0821**
IoT.js-DCO-1.0-Signed-off-by: Jaechul Yang jc08.yang@samsung.com

### CODE REVIEW

**Comment by nova0821** [comment]
@yichoi I added patch for IoT.js-API-reference.md.

File: docs/IoT.js-API-I2C.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+Read one byte from I2C device. `res` contains result as a number.
+
+#### i2c.readBytes(command, length, callback)
+* `command: Number`
+* `length: Number`
+* `callback: Function(err, res)`
+
+Read bytes from I2C device with command.
+`length` is a number of bytes. `res` contains an array of bytes.
```
Comment (chokobole) [inline docs/IoT.js-API-I2C.md]: It it should be the number of bytes?

Comment (nova0821) [inline docs/IoT.js-API-I2C.md]: @chokobole Fixed. Thanks for review.

File: docs/IoT.js-API-I2C.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+## Module: i2c
+
+### Constructor
+
+#### new i2c([address[, options]])
+* `address: Number (Default value is 0x23)`
+* `options: Object`
+
+Option is an object specifying following information:
```
Comment (akosthekiss) [inline docs/IoT.js-API-I2C.md]: `options` instead of Option?

File: docs/IoT.js-API-I2C.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+* `device: String (Default value is /dev/i2c-1)`
+
+Create I2C wire.
+
+### Methods
+
+#### i2c.scan(callback)
```
Comment (akosthekiss) [inline docs/IoT.js-API-I2C.md]: Is this a function provided by the module (e.g., `i2c.scan(...)`) or a function to be called on an object created with the constructor (e.g., `(new i2c()).scan(...)`)? How to differentiate the two types of methods in the doc?

File: docs/IoT.js-API-I2C.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+* `bytes: Array`
+* `callback: Function(err)`
+
+Write bytes to I2C device. `bytes` is an array of numbers.
+
+#### i2c.writeByte(byte, callback)
+* `byte: Number`
+* `callback: Function(err)`
+
+Wirte one byte to I2C device.
```
Comment (akosthekiss) [inline docs/IoT.js-API-I2C.md]: Write

File: docs/IoT.js-API-I2C.md (hunk 2 above)
Comment (nova0821) [inline docs/IoT.js-API-I2C.md]: Fixed.

File: docs/IoT.js-API-I2C.md (hunk 4 above)
Comment (nova0821) [inline docs/IoT.js-API-I2C.md]: Fixed.

File: docs/IoT.js-API-I2C.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+`options` is an object specifying following information:
+* `device: String (Default value is /dev/i2c-1)`
+
+Create I2C object. Following methods can be called with I2C object.
+
+For example,
+```javascript
+var i2c = require('i2c');
+var wire = new i2c(0x23, {device: '/dev/i2c-1'};
```
Comment (akosthekiss) [inline docs/IoT.js-API-I2C.md]: Missing closing parenthesis

Comment (nova0821) [inline docs/IoT.js-API-I2C.md]: @akosthekiss Fixed. Thanks for review.

**Pull request merged**